strawberry/asgi/handlers/http_handler.py

```diff
@@ -54,8 +50,6 @@ async def handle(self, scope: Scope, receive: Receive, send: Send):
             context=context,
         )
 
-        response.headers.raw.extend(sub_response.headers.raw)
```

Comment:
This can be readded, can't it?

Reply:
Good call, didn't mean to delete that. Fixed.